cmd/root_test.go

```diff
@@ -23,6 +23,8 @@ func TestBuildFirewallConfiguration(t *testing.T) {
 			ProxyUID:               expectedProxyUserID,
 			SimulateOnly:           false,
 			UseWaitFlag:            false,
+			BinPath:                "/sbin/iptables",
```

Comment:
Do you think it's a concern that the default behavior is changing to use `/sbin/iptables` instead of just `iptables` anywhere on the PATH?  We're expecting to run this in the linkerd-init container which should have the iptables binary in a predictable place so it's probably okay, but I thought I'd mention it.

Reply:
On the other hand, makeing "iptables" the default instead of `/sbin/iptables` preserves the old behavior and maybe saves us from a gotcha in the future if we change base images or something like that.